ads/google/a4a/utils.js

```diff
@@ -331,3 +335,46 @@ export function additionalDimensions(win, viewportSize) {
           innerWidth,
           innerHeight].join();
 };
+
+/**
+ * Creates amp-analytics element within a4a element using urls specified
+ * with amp-ad closest selector and min 50% visible for 1 sec.
+ * @param {!../../../extensions/amp-a4a/0.1/amp-a4a.AmpA4A} a4a
+ * @param {!../../../src/service/extensions-impl.Extensions} extensions
+ * @param {!Array<string>} urls
```

Comment:
If `urls` is empty, what's the behavior of this function?  In that case, should a block be inserted at all, or should it just skip out?